File: MORM.Java/src/MORM/Java/Models/Venda/Pais.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "PAIS")
public class Pais extends CollectionItem
{
    @CampoAttribute(Campo = "ID_PAIS", Tipo = CampoTipo.Key)
    public Integer Id_Pais;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Req)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Req)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "CD_PAIS", Tipo = CampoTipo.Req)
    public Integer Cd_Pais;
    @CampoAttribute(Campo = "DS_PAIS", Tipo = CampoTipo.Req)
    public String Ds_Pais;
    @CampoAttribute(Campo = "DS_SIGLA", Tipo = CampoTipo.Req)
    public String Ds_Sigla;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Pais.cs
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("PAIS")]
    public class Pais : CollectionItem
    {
        [Campo("ID_PAIS", CampoTipo.Key)]
        public int Id_Pais { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Req)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Req)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_PAIS", CampoTipo.Req)]
        public int Cd_Pais { get; set; }
        [Campo("DS_PAIS", CampoTipo.Req)]
        public string Ds_Pais { get; set; }
        [Campo("DS_SIGLA", CampoTipo.Req)]
        public string Ds_Sigla { get; set; }
    }
}